web/src/pages/tasks/[taskUrlSlug].tsx

```diff
@@ -121,17 +118,7 @@ const TaskPage = (props: Props): ReactElement => {
     if (!taskInRoadmap) {
       return <></>;
     }
-
-    if (userData?.profileData.businessPersona === "FOREIGN") {
-      return <NexusFormationTask task={taskInRoadmap} />;
-    } else {
-      return (
-        <BusinessFormation
-          task={taskInRoadmap}
-          displayContent={props.displayContent.formationDisplayContent}
-        />
-      );
-    }
+    return <BusinessFormation task={taskInRoadmap} displayContent={props.displayContent} />;
```

Comment:
Could we just keep these completely separate? Why combine them all into one Business Formation task that splits off into 2 different flows?